mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -25,12 +25,18 @@
 		<i class="icon-check"></i>
 	</h3>
 
+	<portlet:resourceURL var="resourceUrl"></portlet:resourceURL>
```

Comment:
Use resourceURL identifier and check that identifier in ```serveResource```. If you do not do this, you cannot distinguish one AJAX request from another. Current solution works only if you have one request in the portlet.

See any ```serveResource``` in osb-lcs-portlet as example.